src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -150,6 +150,18 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         ImGui::PopStyleColor();
     }
 
+    // GPU
+    ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+    if (ImGui::BeginTabItem("GPU")) {
+        ImGui::PopStyleColor();
+        ImGui::Checkbox("Hardware Flip", &host.cfg.hardware_flip);
+        if (ImGui::IsItemHovered())
```

Comment:
Check the box to enable flipping from GPU side. Performance and compabilities for commercial games increase.

My wanted tooltip, but need someone to fix the grammar for me

Reply:
ok, is good i things ^^

Reply:
compatibilities sorry. Wait for @1whatleytay to come here and correct the grammar for me first before i can merge

Reply:
Oki :)